codejam-canvas/src/index.js

```diff
@@ -0,0 +1,70 @@
+import './styles/site.scss';
+import {data as img4x4} from './assets/data/4x4';
+import { data as img32x32 } from './assets/data/32x32';
+
+const controls = document.getElementsByClassName('menu__control');
+const types = ['image', '4x4', '32x32'];
+
+let drawPicture = function () {
```

Comment:
Why are there so many 'let's? Do you plan to redeclare these variables?

```suggestion
const drawPicture = function () {
```